### PR TITLE
Updates sdk_automl_video_action_recognition_batch.ipynb

### PR DESCRIPTION
**Pull request by sudarshan-SpringML**
File came in regression log. Unable to download executed notebook. When ran in local, file executed successfully.
**Changes made**:

Update notebook according to template

- [X] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [X] Follow the style and grammar rules outlined in the above notebook template.
- [X] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [X] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [X] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [X] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

### CODE REVIEW

File: notebooks/official/automl/sdk_automl_video_action_recognition_batch.ipynb [hunk 1]
```diff
@@ -29,9 +29,10 @@
         "id": "title"
```
Comment (ivanmkc) [inline notebooks/official/automl/sdk_automl_video_action_recognition_batch.ipynb:29]: <p>For <code>Create a Vertex AI&nbsp;Dataset resource.</code></p><p> </p><p>Please remove formatting. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (ivanmkc) [inline notebooks/official/automl/sdk_automl_video_action_recognition_batch.ipynb:29]: <p>See the boilerplate reduction instructions. This should all be removed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (ivanmkc) [inline notebooks/official/automl/sdk_automl_video_action_recognition_batch.ipynb:29]: <p>Please review instructions. This all needs to be removed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

**Review by ivanmkc — APPROVED** [review]
LGTM

**Comment by sudarshan-SpringML** [comment]
Error: Step #5: TIMEOUT
Step #5: ERROR: context deadline exceeded

**Comment by sudarshan-SpringML** [comment]
Executed notebook cannot be downloaded, No such URL Error

**Pull request merged**